README.md

```diff
@@ -207,8 +208,28 @@ If you have not yet cloned the Harmony repository, run
 ```
 $ git clone https://github.com/nasa/harmony.git
 ```
+Ensure `envsubst` is installed on your system.
 
-Ensure envsubst is installed on system.
+For Mac, run
+```
+brew install envsubst
```

Comment:
I happened to have tried this a bit ago and it didn't work. You need to run `brew install gettext`.

Reply:
Doh! You're right.

Reply:
Fixedx.